framework/doc/content/source/postprocessors/InterfaceIntegralVariableAveragePostprocessor.md

```diff
@@ -0,0 +1,21 @@
+# InterfaceIntegralVariableAveragePostprocessor
+
+## IMPORTANT NOTE
+*** Any derived class of InterfaceUserObject does not support yet getMaterialProperty, please use auxvariables ***
+
+## Description
+!syntax description /Postprocessors/InterfaceIntegralVariableAveragePostprocessor
+
+This post-processor is used to compute an average value across an interface. The type of average to compute is selected by changing the `average_type` parameter. If the parameter `average_type` is omitted it default on computing the average value between the two input variables across the interface.
+The kind of average to compute are describe in more details in  [InterfaceAverageTools](/InterfaceAverageTools.md).
```

Comment:
The kind of average to compute are describe -> The __various types__ of __averages__ are __described__...